webpack/components/SearchResult.js

```diff
@@ -164,7 +164,7 @@ export default class SearchResult extends Component {
           var selectedResponseSet = this.props.responseSets.find((r) => r.id == this.props.selectedResponseSetId);
           return (
             <div className="panel-body panel-body-form-question">
-              <span className="selected-response-set">Response Set: {(selectedResponseSet && selectedResponseSet.name) || '(None)'}</span>
+              <span className="selected-response-set" aria-label={`Selected Response set for Question ${result.content}`}>Response Set: {(selectedResponseSet && selectedResponseSet.name) || '(None)'}</span>
```

Comment:
Getting cannot read .content of undefined in some places now. need to check that object is defined, also it might be result.name or some other logic like when the name is used in the widget title like:
const name = result.content ? result.content: result.name
because questions sometimes have .content and sometimes have .name depending on the source